_docs/_latest/develop/website.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Edit & Build Website
+order: 003
+---
+
+#  {{ page.title }}
+
+The instructions below are aimed at developers using Linux, particularly Debian
```

Comment:
I would also add a blurb out the motivation of this page, that says something to the effect that:
```
This pages shows you how to install and run the website server lcoally, so that you can test your documentation changes before merging.
```